src/status/views.py

```diff
@@ -26,7 +26,8 @@ def get_last_calibration_time():
     """Returns datetime string of last calibration time"""
     sdr.connect()
     if sdr.is_available and sdr.radio.sensor_calibration:
-        return sdr.radio.sensor_calibration.calibration_datetime
+        cal_datetime = sdr.radio.sensor_calibration.calibration_datetime
```

Comment:
I'll update `sdr_calibrator` to use the new datetime format as well (I think it is still worthwhile having this in there though)